### PR TITLE
feat: [PL-25319]: Account Basic Role UI changes

### PR DESCRIPTION
**Pull request by reetika04**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by reetika04** [comment]
trigger rebase

File: src/modules/20-rbac/components/RoleBindingsList/RoleBindingsList.tsx [hunk 1]
```diff
@@ -62,6 +65,13 @@ const RoleBindingTag = ({ roleAssignment }: { roleAssignment: RoleBinding }): Re
 }
 
 const RoleBindingsList: React.FC<RoleBindingsListProps> = ({ data, length = data?.length, showNoData = false }) => {
+  const { ACCOUNT_BASIC_ROLE } = useFeatureFlags()
+  const { accountId, orgIdentifier, projectIdentifier } = useParams<ProjectPathProps>()
+  const scope = getScopeFromDTO({ accountIdentifier: accountId, orgIdentifier, projectIdentifier })
+  data = data?.filter(
```
Comment (avinashmadhwani02) [inline src/modules/20-rbac/components/RoleBindingsList/RoleBindingsList.tsx:71]: Are you not getting any errors here? I think we can't change the prop value.

**Review by avinashmadhwani02 — DISMISSED** [review]

File: src/modules/20-rbac/components/RoleBindingsList/RoleBindingsList.tsx (hunk 1 above)
Comment (upadhyay-an) [reply src/modules/20-rbac/components/RoleBindingsList/RoleBindingsList.tsx:71]: Also would the default value assignment to length prop is still valid after  the assignment?

**Review by upadhyay-an — DISMISSED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Comment by reetika04** [comment]
trigger cypress

**Review by avinashmadhwani02 — APPROVED** [review]

**Comment by reetika04** [comment]
retrigger coverage

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Pull request merged**